packages/commonjs/test/fixtures/function/esm-mixed-exports-function-default/esm-constructor.js

```diff
@@ -0,0 +1,9 @@
+export default function Foo() {
```

Comment:
Consider adding a couple of arguments to function Foo to verify the `f.apply(this, arguments)` trampoline.